examples/word_of_day.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const libingester = require('libingester');
+const mustache = require('mustache');
+const rp = require('request-promise');
+const template = require('./word_of_day_template');
```

Comment:
@sgnn7 we use this convention everywhere else, IMO it's totally fine since it's just a shortcut to `fs.readFileSync`